Update categories

krischulz opened the pull request.

Commit:
Updated initial test categories

Commit:
tied in the back end category code to the front end grouping code

Commit:
Begin & Execute button style update

arscan committed:
hide groups that have no visible tests due to filters

arscan committed:
fix execute button hover color

arscan commented:
@krischulz and @andrequina I put in the hide group feature in here.  I also removed some code that was causing the execute button to lose it's text on hover.  So you might want to pull this down and check it out before merging.

krischulz commented:
I fixed both hover modes. I’ll need to update to git.
Regards,
Kris

## 

Kris Schulz
Healthcare UX Design Lead
K84C – Health Information Technology
781-271-2072 – Office
203-394-2112 – Mobile

[cid:1E30A463-4412-42B1-8804-B855BF791978]https://www.facebook.com/MITREcorp[cid:F2A2EFBB-622D-427A-B393-363A484C9B8D]https://www.linkedin.com/company/mitre[cid:4B4D2E69-ACD3-40EA-AA5C-2D6EA7C7B3C5]https://twitter.com/MITREcorp[cid:C62D3664-CB9E-4F20-83DE-A2405732522F]https://www.youtube.com/user/mitrecorp[cid:3E09BAFF-6DCE-4925-BAAD-F04737C356F6]https://plus.google.com/+MitreOrgFFRDCs/posts

[cid:35986943-6982-4C2B-A1DB-1C4C4E59313F]http://www.mitre.org/

From: Rob Scanlon <notifications@github.com<mailto:notifications@github.com>>
Reply-To: fhir-crucible/crucible <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Friday, February 26, 2016 at 3:56 PM
To: fhir-crucible/crucible <crucible@noreply.github.com<mailto:crucible@noreply.github.com>>
Cc: "Schulz, Kris" <kwschulz@mitre.org<mailto:kwschulz@mitre.org>>
Subject: Re: [crucible] Update categories (#129)

@krischulzhttps://github.com/krischulz and @andrequinahttps://github.com/andrequina I put in the hide group feature in here. I also removed some code that was causing the execute button to lose it's text on hover. So you might want to pull this down and check it out before merging.

—
Reply to this email directly or view it on GitHubhttps://github.com/fhir-crucible/crucible/pull/129#issuecomment-189480305.

krischulz committed:
hover behavior fixed

krischulz committed:
New button hover fix
